packages/components/inputs/select-input/src/select-input.tsx

```diff
@@ -324,11 +324,10 @@ export type TSelectInputProps = {
 
 const defaultProps: Pick<
   TSelectInputProps,
-  'maxMenuHeight' | 'menuPortalZIndex' | 'controlShouldRenderValue' | 'options'
+  'maxMenuHeight' | 'menuPortalZIndex' | 'options'
 > = {
   maxMenuHeight: 220,
   menuPortalZIndex: 1,
-  controlShouldRenderValue: true,
```

Comment:
I guess technically this is a breaking change, but this was added only to fix the ui bug.